### ISSUE TITLE
LFU to be implemented in Java

### ISSUE THREAD

**Issue by sbshah97** [WebClub-NITK/LFU#1]
* Kindly assign this Issue to @hrily and @salman-bhai. 
* Will send a PR in a maximum of two weeks!
* Also if possible can a CONTRIBUTING.md file be made?

**Comment by chinmaydd** [comment]
Sounds good. I will try adding some contributing guidelines soon. Although, I am not really sure what you are looking for. The README does contain instructions on how to go about implementing. As for contributions is concerned, if your implementation is in a language `X` make sure to create a branch for `X` and push to it.

**Comment by sbshah97** [comment]
Yeah, also if possible can you and @mohitreddy1996 push your code from the forked repository to the main repository!

**Comment by mohitreddy1996** [comment]
A little bit of code refactoring to be done. Will push the code to the mail repository once done.

**Comment by chinmaydd** [comment]
> your code from the forked repository

It's still a bit incomplete :cry: Will upload once I am done.

**Comment by mohitreddy1996** [comment]
Test cases added. C++ Code added with appropriate test cases using CppUnit added on branch C++.
